case_studies/galaxy_clustering/README.md

```diff
@@ -37,24 +37,23 @@ This project is being conducted under the Undergraduate Research Program in Stat
+*The task may be seen as creating a subimage inside the original image.* We consider a cluster with a center. For the center, it would have a bounding box and all galaxies within the bounding box should perform differently. To make the image look more real, the center should be in another bounding box of the overall image encompassing the region between 25% and 75% of its side length. The center's own bounding box shall have side length no larger than 50% of the image side length.
 
-In our cluster, we first calcuate the area of the bounding box and convert it to the equal number of titles. Then we times it with maximum number of sources within a tile and randomly pick 80% out of it (gives a slightly larger value than the average number of sources per tile). The fluxes/params for the galaxy so far follows the identical implementation for normal galaxies. In the future, we aim to use the redmapper paper to further modify the flux and params. 
+In our cluster, we first calcuate the area of the bounding box and convert it to the equal number of titles. Then we times it with maximum number of sources within a tile and randomly pick 80% out of it (gives a slightly larger value than the average number of sources per tile). The fluxes/params for the galaxy so far follows the identical implementation for normal galaxies. In the future, we aim to use the redmapper paper to further modify the flux and params.
 
-So far, the only change within the cluster was to increase the rate of galaxies. That is, we use the Possion distribution to first sample the number of sources based on number of tiles and the average number of sources within tiles. A predetermined probability of a source being a galaxy determines how many galaxies will ultimately be present within the image. 
+So far, the only change within the cluster was to increase the rate of galaxies. That is, we use the Possion distribution to first sample the number of sources based on number of tiles and the average number of sources within tiles. A predetermined probability of a source being a galaxy determines how many galaxies will ultimately be present within the image.
 
-For implementation details, we generate location, fluxes, params, and types for the cluster first. Then we follow the normal procedure of generating sources. In the end, we stack two sets of vectors together to have the final values. 
+For implementation details, we generate location, fluxes, params, and types for the cluster first. Then we follow the normal procedure of generating sources. In the end, we stack two sets of vectors together to have the final values.
```

Comment:
Better to leave the galaxy_clustering case study unmodified in this PR. (Even changing whitespace can cause merge conflicts and changes the git history of the file.)